dev-docs/bidders/ix.md

```diff
@@ -97,23 +97,17 @@ In this configuration Prebid.js calls Index directly from the browser using our
    * [Download Prebid.js](https://docs.prebid.org/download.html) from the Prebid site to use the standard compiled binary that Prebid includes in the download process and select **Index Exchange** as an adapter.
    * Build it on your own from the source code by following the instructions in [Prebid.js project README](https://github.com/prebid/Prebid.js/blob/master/README.md#build-optimization). If you use this method, you will need to include several modules in your build process. See the [Index modules to include in your build process](#modules-to-include-in-your-build-process) section below.
 2. Define the Index-specific parameters at the bidder level which include adding `ix` as the bidder and the `siteId`. For Index's bidder-specific parameters, see the [Bid request parameters](#bid-request-parameters) section below. <br />
-**Example:**
-
-```javascript
-{
-    bidder: 'ix',
-    params: {
+   ```javascript
+        {
```

Comment:
The formatting is a little awkward here. Consider aligning the braces?

Reply:
thanks for looking. Updated